app/components/shared/navbar_component/navbar_component.html.haml

```diff
@@ -13,4 +13,12 @@
       %label.govuk-label.navigation_heading
```

Comment:
nothing to do with this PR, but i am going to remove these labels. there is no point to them and you should only have li as direct children of ul/ol

Reply:
ill comment in my subsequent PR